exonum/src/blockchain/schema.rs

```diff
@@ -95,38 +95,35 @@ impl TxLocation {
 /// Indices defined by this schema are present in the blockchain regardless of
 /// the deployed services and store general-purpose information, such as
 /// committed transactions.
-#[derive(Debug)]
+#[derive(Debug, Clone, Copy)]
 pub struct Schema<T> {
     access: T,
```

Comment:
Nit: I think a small note for developers why we don't use a field-based schema here will be nice.